shared_model/schema/queries.proto

```diff
@@ -103,4 +103,7 @@ message Query {
 message BlocksQuery {
   QueryPayloadMeta meta = 1;
   Signature signature = 2;
+  oneof opt_height {
```

Comment:
It does not seem intuitive that a single parameter changes the logic of the query - from a continuous stream of new blocks to returning a single block. Why not to create a non-streaming query that will return a single block?